Jenkinsfile

```diff
@@ -39,16 +39,24 @@ node {
     }
 }
 
+// The sleep command in the parallel blocks is required to prevent triggering
+// the deployment workflow with what Jenkins sees as the same values.
+// 
+// Information on troubleshooting can be found in slack:
+// https://hypothes-is.slack.com/archives/CR3E3S7K8/p1625041642057400 
+//
+// If we decide to extend the parallel block to deploy to another environment 
+// or region a sleep command should be included before calling deployApp.
```

Comment:
Comment detailing why we are using the `sleep x` command in the parallel blocks

Reply:
The comment has been removed in favour of a smaller comment directly above the sleep command

> // Workaround to ensure all parallel builds happen. See https://hypothes-is.slack.com/archives/CR3E3S7K8/p1625041642057400